dags/utils/tbv.py

```diff
@@ -0,0 +1,24 @@
+"""Utility functions for launching telemetry-batch-view jobs"""
+
+
+def tbv_envvar(klass, options, other={}):
+    """Set up environment variables for telemetry-batch-view jobs.
+
+    The command line interface can read options from the environment. All
+    environment variables must be prefixed by `TBV_`. For example, a class in
+    telemetry-batch-view taking a `--date` option can use `TBV_DATE` instead.
+
+    :klass string:      name of the class in telemetry-batch-view
+    :options dict:      environment variables to prefix
+    :other dict:        environment variables to pass through
+
+    :returns: a dictionary that contains properly prefixed class and options
+    """
+    prefixed_options = {
```

Comment:
Perhaps we could use two underscore to indicate a dash?

Reply:
In the case of mozetl, the envvar wrapper and click definition will both downcase the dash to an underscore so there isn't a mismatch.

 Mixing underscores and dashes is inconsistent anyways. We should just enforce the convention of using dashes in the argument list by throwing an exception and letting CI/local testing catch it.

Reply:
> We should just enforce the convention of using dashes in the argument list by throwing an exception and letting CI/local testing catch it.

in `tbv_envvars` or in [telemetry-batch-view](https://github.com/mozilla/telemetry-batch-view)? I can throw errors on `_`s in `tbv_envvars`, but the real place to catch this imo would be in [telemetry-batch-view](https://github.com/mozilla/telemetry-batch-view).

Reply:
Ideally, the cli tooling would tell you that your arguments are ill-formatted, but you would have to extend the functionality of scallop to catch the use of underscores in tbv. It might be more overhead than it's worth though. 

It should be fine as long as something automated catches the error before it reaches production.